cves/CVE-2013-0861.yml

```diff
@@ -344,7 +380,15 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    From the commit messages I say, it was clear that the developers working on
```

Comment:
Consider removing the use of "I", e.g. "From the commit messages, ..."

Reply:
Removed personal pronoun.